### PR TITLE
Update extends version

### PR DESCRIPTION
**Pull request by nickcall**

### CODE REVIEW

File: toolkits/nature/packages/nature-article/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 0.4.1 (2020-01-27)
+	* Update the extends package version of glbal article from 7.2.0 to 9.3.0
```
Comment (saramorueta) [inline toolkits/nature/packages/nature-article/HISTORY.md]: missing 'o'

File: toolkits/nature/packages/nature-article/package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "@springernature/nature-article",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (saramorueta) [inline toolkits/nature/packages/nature-article/package.json]: I'd say this is a major version because we might be including a lot of new stuff right?

Comment (nickcall) [reply toolkits/nature/packages/nature-article/package.json]: yeah I guess, will up date. thanks.

Comment (davidpauljunior) [reply toolkits/nature/packages/nature-article/package.json]: You don't really need to major version if it's still on version 0 (development).
If this is used in production then it should _already_ be a major version.

Comment (nickcall) [reply toolkits/nature/packages/nature-article/package.json]: Have pushed it up to 0.5.0, is that OK ?

Comment (nickcall) [reply toolkits/nature/packages/nature-article/package.json]: I should really have done this as part of the previous PR

**Comment by davidpauljunior** [comment]
In this PR it doesn't really matter in my opinion.
However, if this article component is used in production and changes could result in breakages then a major version should be released (same for the global article).
That can be done in separate PRs.

**Pull request merged**